app/__init__.py

```diff
@@ -4,6 +4,13 @@
 import sys
 from datetime import timedelta
 
+from flask_login import LoginManager
+from structlog import configure
+from structlog import get_logger
+from structlog.processors import JSONRenderer
+from structlog.stdlib import LoggerFactory
+from structlog.threadlocal import wrap_dict
+
 from app import settings
 from app.authentication.authenticator import Authenticator
```

Comment:
This references a deleted class which causes `ImportError: cannot import name 'Authenticator'`